packages/cli/internal/pkg/cli/configure_describe.go

```diff
@@ -43,7 +43,7 @@ func BuildDescribeShowCommand() *cobra.Command {
 		RunE: runCmdE(func(cmd *cobra.Command, args []string) error {
 			opts, err := newConfigureDescribeContextOpts()
 			if err != nil {
-				return err
+				return clierror.New("configure describe", nil, err)
```

Comment:
Please use a string constant to eliminate duplicated string literal. Can this or similar string be obtained from Cobra object?